conan/cli/commands/graph.py

```diff
@@ -16,7 +16,7 @@
 @conan_command(group="Consumer")
 def graph(conan_api, parser, *args):
     """
-    Computes a dependency graph, without  installing or building the binaries
+    Compute a dependency graph, without  installing or building the binaries.
```

Comment:
I am not sure the docstrings should be plural/first person.

The command ``conan graph`` **computes** a dependency graph...

Reply:
Python's stdlib uses the imperative tense, such as:

```py
def isinstance(x, A_tuple):
    """
    Return whether an object is an instance of a class or of a subclass thereof.
    """
    ...
```

I'm happy to go with whatever you prefer, no strong opinions either way